Sources/XMTPiOS/Client.swift

```diff
@@ -75,7 +75,7 @@ public struct ClientOptions {
 		enableV3: Bool = false,
 		encryptionKey: Data? = nil,
 		dbDirectory: String? = nil,
-		historySyncUrl: String? = nil
+		historySyncUrl: String = "https://message-history.dev.ephemera.network/"
```

Comment:
@neekolas just confirming this is the url we're using now.

Reply:
Is there a prod network as well? Should this correlate with the env the user is using? FYI @jhaaaa

Reply:
Yes, that’s the right URL. There is a production version deployed as well at https://message-history.production.ephemera.network/